.github/workflows/main.yml

```diff
@@ -55,11 +57,17 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
 
+      - name: Use Pipfile.lock on locked version of Python
+        # Keep version in sync with scripts/update-pipfile-lock/Dockerfile
+        if: ${{ matrix.python-version }} == "3.9"
+        run: |
+          echo "PIPENV_ARGS=--ignore-pipfile" >> $GITHUB_ENV
+
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
           python -m pip install --upgrade pipenv
-          pipenv install --system --dev --ignore-pipfile
+          pipenv install --system --dev $PIPENV_ARGS
```

Comment:
If we have to lock deps for all versions of Python except 3.9, this will increase the duration of the CI steps correct ? Anyway, it looks like we don't have many solutions.

Reply:
Yes, and it would need one lock file per Python version.